config/settings.py

```diff
@@ -13,28 +16,33 @@
 
 ALLOWED_HOSTS = ["*"]
 
-
 # Application definition
-AUTHENTICATION_BACKENDS = ("app.backends.CaseInsensitiveModelBackend",)
+AUTHENTICATION_BACKENDS = ("users.backends.CaseInsensitiveModelBackend",)
 
 INSTALLED_APPS = [
-    "app.apps.AppConfig",
     "django.contrib.admin",
     "django.contrib.auth",
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django.contrib.humanize",
+    "django.contrib.sites",
+
     "mptt",
     "rest_framework",
     "rest_framework.authtoken",
     "rest_auth",
-    "django.contrib.sites",
+
     "allauth",
     "allauth.account",
     "allauth.socialaccount",
     "rest_auth.registration",
+
+    "entries.apps.EntriesConfig",
```

Comment:
There's no need to point to specific config as Django handles that just fine.
```suggestion
    "entries",
```

Reply:
It is recommended to use config  i read that in the django 2 scoope

Reply:
Ok, docs prefer it that way too.